2025-08-17 11:04:38.889 [warning] Dir "/home/shaiksahil0911/.pixi/envs" is not watchable (directory does not exist)
2025-08-17 11:04:38.889 [warning] Failed to check if /home/shaiksahil0911/venv/bin/python is an executable [Error: ENOENT: no such file or directory, lstat '/home/shaiksahil0911/venv/bin/python'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'lstat',
  path: '/home/shaiksahil0911/venv/bin/python'
}
2025-08-17 11:04:38.893 [warning] Identifier for virt-virtualenv failed to identify /home/shaiksahil0911/venv/bin/python [Error: ENOENT: no such file or directory, scandir '/home/shaiksahil0911/venv/bin'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/shaiksahil0911/venv/bin'
}
2025-08-17 11:04:39.070 [warning] Failed to check if /home/shaiksahil0911/venv/bin/python is an executable [Error: ENOENT: no such file or directory, lstat '/home/shaiksahil0911/venv/bin/python'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'lstat',
  path: '/home/shaiksahil0911/venv/bin/python'
}
2025-08-17 11:04:39.072 [warning] Identifier for virt-virtualenv failed to identify /home/shaiksahil0911/venv/bin/python [Error: ENOENT: no such file or directory, scandir '/home/shaiksahil0911/venv/bin'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/shaiksahil0911/venv/bin'
}
2025-08-17 11:04:39.092 [info] > ./venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py
2025-08-17 11:04:39.116 [error] [Error: Command failed: /home/shaiksahil0911/venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py
/bin/sh: line 1: /home/shaiksahil0911/venv/bin/python: No such file or directory

	at genericNodeError (node:internal/errors:984:15)
	at wrappedFn (node:internal/errors:538:14)
	at ChildProcess.exithandler (node:child_process:422:12)
	at ChildProcess.emit (node:events:536:35)
	at maybeClose (node:internal/child_process:1104:16)
	at Socket.<anonymous> (node:internal/child_process:456:11)
	at Socket.emit (node:events:524:28)
	at Pipe.<anonymous> (node:net:343:12)] {
  code: 127,
  killed: false,
  signal: null,
  cmd: '/home/shaiksahil0911/venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py'
}
2025-08-17 11:04:40.778 [info] > pyenv which python
2025-08-17 11:04:40.778 [info] cwd: .
2025-08-17 11:04:43.047 [info] > ./venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py
2025-08-17 11:04:43.244 [error] Error: Command failed: /home/shaiksahil0911/venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py
/bin/sh: line 1: /home/shaiksahil0911/venv/bin/python: No such file or directory

    at genericNodeError (node:internal/errors:984:15)
    at wrappedFn (node:internal/errors:538:14)
    at ChildProcess.exithandler (node:child_process:422:12)
    at ChildProcess.emit (node:events:536:35)
    at maybeClose (node:internal/child_process:1104:16)
    at Socket.<anonymous> (node:internal/child_process:456:11)
    at Socket.emit (node:events:524:28)
    at Pipe.<anonymous> (node:net:343:12) {
  code: 127,
  killed: false,
  signal: null,
  cmd: '/home/shaiksahil0911/venv/bin/python -I /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/get_output_via_markers.py /google/devshell/editor/code-oss-for-cloud-shell/extensions/ms-python.python/python_files/interpreterInfo.py'
}
2025-08-17 11:04:45.071 [info] Starting Jedi language server for shaiksahil0911.
